db/employees.json

```diff
@@ -0,0 +1,7 @@
+{
+  "employee_id": "3",
+  "first_name": "Albert",
+  "last_name": "Einstein",
+  "added_date": "2018-05-20T00:00:00.000",
+  "employment_status": "1"
+}
```

Comment:
Lets update this `json` model with camelCase names. Camel case names are json convention too.

Reply:
Should I leave the employeeId or is it better to remove it?

Reply:
keep it because it is a model of what can be there. You can put a comment saying that this value is omitted on POSTs